internal/configconverter/normalize_gcp.go

```diff
@@ -0,0 +1,78 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configconverter
+
+import (
+	"context"
+	"fmt"
+	"go.opentelemetry.io/collector/confmap"
+	"log"
+)
+
+type NormalizeGcp struct{}
+
+func (NormalizeGcp) Convert(_ context.Context, in *confmap.Conf) error {
+	if in == nil {
+		return fmt.Errorf("cannot NormalizeGcp on nil *confmap.Conf")
```

Comment:
I'm not sure the precedent for this is correct and simply returning nil would probably be better.

Reply:
who are we warning and to what end? Converters are used by the core service: https://github.com/open-telemetry/opentelemetry-collector/blob/776c6b38443f431915493febc3a283961a97d0db/confmap/resolver.go#L178

Reply:
thanks, missed update